tcl/response.tcl

```diff
@@ -178,7 +179,26 @@ proc qc::response2tson {} {
             switch $key {
                 record {
                     foreach {name values} $value {
-                        lappend record_objects $name [list object valid [dict get $values valid] value [list string [dict get $values value]] message [list string [dict get $values message]]]
+                        # Check if this is a sensitive column (should not be echo'ing back the value to the client)
+                        set sensitive_column false
+                        if { [qc::db_column_exists $name] } {
+                            set column $name
+                            set tables [qc::db_column_table $name]
+                            foreach table $tables {
+                                if { [qc::db_column_type $table $column] in [list password card_number] } {
```

Comment:
I think the same ````password``` and ```card_number``` (domain) can still be applied to those too.

Might need to do this in the future though if we come across any more ... or I can make that provision now if you like